packages/shared/lib/models/Proxy.ts

```diff
@@ -4,12 +4,25 @@ import type { BasicApiCredentials, ApiKeyCredentials, AppCredentials, TbaCredent
 import type { Connection } from './Connection.js';
 import type { Template as ProviderTemplate } from '@nangohq/types';
 
+export interface File {
```

Comment:
I'm surprised it's not provided by express?

Reply:
it is provided by the multer middleware but I didn't want to add it as a dependency of shared. https://expressjs.com/en/resources/middleware/multer.html

Reply:
I think it's fine if it's a dev deps (we tested with `@nangohq/*`) as you want ☺️